Mathlib/Algebra/Associated.lean

```diff
@@ -728,6 +728,13 @@ theorem Associated.of_pow_associated_of_prime' [CancelCommMonoidWithZero α] {p
   (h.symm.of_pow_associated_of_prime hp₂ hp₁ hk₂).symm
 #align associated.of_pow_associated_of_prime' Associated.of_pow_associated_of_prime'
 
+lemma Prime.coprime_iff_not_dvd' [CancelCommMonoidWithZero α] {p n : α} (hp : Prime p) :
```

Comment:
Maybe you could also add a pointer here to the other lemma, the one without the `'`?

Reply:
Agreed, done!